src/content/blog/2022/06/15/react-labs-what-we-have-been-working-on-june-2022.md

```diff
@@ -72,8 +72,8 @@ We are working on a new version for the Interaction Tracing API (tentatively cal
 
 ## New React Docs {/*new-react-docs*/}
 
-Last year, we announced the [beta version](https://beta.reactjs.org/) of the new React documentation website. The new learning materials teach Hooks first and has new diagrams, illustrations, as well as many interactive examples and challenges. We took a break from that work to focus on the React 18 release, but now that React 18 is out, we’re actively working to finish and ship the new documentation.
+Last year, we announced the [beta version](https://react.dev/) of the new React documentation website. The new learning materials teach Hooks first and has new diagrams, illustrations, as well as many interactive examples and challenges. We took a break from that work to focus on the React 18 release, but now that React 18 is out, we’re actively working to finish and ship the new documentation.
```

Comment:
Here, too: "we announced the beta version of the new React documentation website ([later shipped as react.dev](https://react.dev/blog/2023/03/16/introducing-react-dev)) ..."

Reply:
done.